src/components/onboarding/wizard.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react"
+import Button from "../buttons/default"
+
+interface Props {
+  stepComponents: Array<React.ComponentClass<any>>
```

Comment:
Out of curiosity, have you considered using `children` for this?

Reply:
We're passing in an array of component classes, not instances so `children` wouldn't work in this case. The idea is to simplify the wizard API, all you need to pass is an array of classes where the props inherit from `StepProps`.

Example:

```typescript
 <Wizard stepComponents={[CollectorIntent, Artists]} />
```

The type should be more specific though `Array<React.ComponentClass<StepProps>>` and we should rename this property as the name is misleading.

Reply:
Makes sense 👍